esp8266/modpybuart.c

```diff
@@ -36,32 +36,40 @@
 #include "py/mperrno.h"
 #include "modpyb.h"
 
-// baudrate is currently fixed to this value
-#define UART_BAUDRATE (115200)
+// UartDev is defined and initialized in rom code.
+extern UartDevice UartDev;
 
 typedef struct _pyb_uart_obj_t {
     mp_obj_base_t base;
     uint8_t uart_id;
+    uint32_t baudrate;
+    uint8_t bits;
+    uint8_t parity;
+    uint8_t stop;
```

Comment:
Please reorder so the 4 uint8_t's are together to improved the packing of the struct.